tests/pipeline/test_blaze.py

```diff
@@ -1128,3 +1131,82 @@ def test_novel_deltas_macro(self):
                 window_length=3,
                 compute_fn=op.itemgetter(-1),
             )
+
+    def test_ignore_nonexistent_delta(self):
```

Comment:
we can always parameterize on the context manager, and in one case use `zipline.utils.context_tricks.nop_context`